apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/QueryTest.cs

```diff
@@ -491,5 +491,23 @@ public async Task OrQueriesAsync()
             var snapshot = await query.GetSnapshotAsync();
             Assert.Equal(2, snapshot.Count);
         }
+
+        [SkippableFact]
+        public async Task MultipleInequalitiesAsync()
```

Comment:
I'm surprised we don't have conformance tests for these. Firestore conformance tests seem to be very exhaustive.